.github/workflows/custom-test.yml

```diff
@@ -211,3 +211,35 @@ jobs:
               --certificate-identity $IDENTITY \
               --certificate-oidc-issuer https://token.actions.githubusercontent.com \
               artifact
+
+  sigstore-rs:
+    runs-on: ubuntu-latest
+    needs: [sigstore-python]
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          repository: "sigstore/sigstore-rs"
+          ref: "7769a0c81d53b0174d4e1558108844be439705a4"
+          persist-credentials: false
+
+      - name: tweak sigstore-rs to use the published TUF repository
+        run: |
+          sed -ie "s#https://tuf-repo-cdn\.sigstore\.dev#$METADATA_URL#" src/trust/sigstore/constants.rs
```

Comment:
if we need to do a signing event, we'll want to replace trust_root/prod/root.json in the sigstore-rs sources with the new root as well (like e.g. sigstore-go test already does)

Reply:
this should now work: the branch of sigstore-rs contains the v12 root